node/network/gossip-support/src/lib.rs

```diff
@@ -460,6 +488,7 @@ async fn update_gossip_topology<Context>(
 	our_index: usize,
 	authorities: Vec<AuthorityDiscoveryId>,
 	relay_parent: Hash,
+	session_index: SessionIndex,
```

Comment:
we can also pass `random_seed` from session info instead of calling babe runtime API